python/dgl/graphbolt/impl/ondisk_dataset.py

```diff
@@ -475,6 +481,21 @@ def _init_tvt_set(
             ret = ItemSetDict(data)
         return ret
 
+    def _init_all_nodes_set(self) -> Union[ItemSet, ItemSetDict]:
+        if not isinstance(self._graph, CSCSamplingGraph):
+            return None
+        num_nodes = self._graph.num_nodes
+        if isinstance(num_nodes, int):
+            return ItemSet(num_nodes)
+        else:
+            if len(num_nodes) == 1:
+                return ItemSet(next(iter(num_nodes.values())))
```

Comment:
is it a `dict` even length is 1? If yes, it should be instantiated as `ItemSetDict`.

Reply:
and that's what i do in previous ones